src/huggingface_hub/file_download.py

```diff
@@ -468,3 +468,65 @@ def _resumable_file_manager() -> "io.BufferedWriter":
                 json.dump(meta, meta_file)
 
     return cache_path
+
+
+def hf_hub_download(
+    repo_id: str,
+    filename: str,
+    subfolder: Optional[str] = None,
+    repo_type: Optional[str] = None,
+    revision: Optional[str] = None,
+    library_name: Optional[str] = None,
+    library_version: Optional[str] = None,
+    cache_dir: Union[str, Path, None] = None,
+    user_agent: Union[Dict, str, None] = None,
+    force_download=False,
+    force_filename: Optional[str] = None,
+    proxies=None,
+    etag_timeout=10,
+    resume_download=False,
+    use_auth_token: Union[bool, str, None] = None,
+    local_files_only=False,
+):
+    """
+    Resolve a model identifier, a file name, and an optional revision id, to a huggingface.co file distributed through
+    Cloudfront (a Content Delivery Network, or CDN) for large files (more than a few MBs).
+
+    The file is cached locally: look for the corresponding file in the local cache. If it's not there,
+    download it. Then return the path to the cached file.
+
+    Cloudfront is replicated over the globe so downloads are way faster for the end user (and it also lowers our
+    bandwidth costs).
```

Comment:
```suggestion
    Cloudfront is replicated over the globe so downloads are way faster for the end user.
```

at this stage we don't care about the $ anymore 😂